kpatch/kpatch

```diff
@@ -74,14 +74,14 @@ __find_module () {
 	MODULE="$1"
 	[[ -f "$MODULE" ]] && return
 
-	MODULE=$INSTALLDIR/$(uname -r)/"$1"
+	MODULE="$INSTALLDIR/$(uname -r)/$1"
```

Comment:
The `$1` needs to be quoted.

Reply:
As above, quoted by the surrounding quotes.